docs/experiments-wapo.md

```diff
@@ -25,8 +25,8 @@ After indexing is done, you should be able to perform a retrieval run:
 ```
 sh target/appassembler/bin/SearchCollection \
 -topicreader Trec -index lucene-index.wapo.pos+docvectors \
--bm25 -topics src/main/resources/topics-and-qrels/topics.web.251-300.txt  \
--output run.wapo.251-300.bm25.txt
+-bm25 -topics src/main/resources/topics-and-qrels/topics.451-550.txt  \
```

Comment:
Core18 uses the old topics? I searched online but did not find the topics.

Reply:
I'm not sure. I don't have enough context for this. I changed the topics file because `topics.web.251-300.txt` doesn't contain the tag `<top>` required in `TrecTopicReader`.

Reply:
I do not find the topics https://trec-core.github.io/2018/ 
@Kytabyte you are working on Core18, right? Could you please put the query topics in the repo?

Reply:
Sure. When I first wrote this doc they hadn't published the topics for Core18. Now they have. The topic is old topics for Core17, `topics.core17.txt`, plus `topics.801-850.txt`. They're both in the repo.

Reply:
This collection is brand new for TREC 2018 CORE, so using topics 451-550 is fine for now.

Reply:
Okay...in the mailing list say the new topics are numbered 801-825, I thought that was the one we have...